applications/cambiatus.md

```diff
@@ -211,13 +215,13 @@ Registered Address: PO Box 2510, Grand Cayman KY1-1104, Cayman Islands
 
 
 ### Team Experience
-Our team has been building on EOSIO for 4 years now. We've done our pilot in a community in Costa Rica. We've been part of every major announcement since before Block.one released version 1.0. Since then we've deployed and maintained our own private network, developed several features into our app, including smart contract, backend, infrastructure, wallet integrations, frontend and finally our own wallet inside our app.
+Our team has been building on EOSIO for 4 years now. We've done our pilot in a [community in Costa Rica](https://www.cambiatus.com/verdes), which is still ongoing. We've been part of every major announcement since before Block.one released version 1.0. Since then we've deployed and maintained our own private network, developed several features into our app, including [smart contract](https://github.com/cambiatus/contracts), [backend](https://github.com/cambiatus/backend), [infrastructure](https://github.com/cambiatus/backend#nginx-setup), wallet integrations, [frontend](https://github.com/cambiatus/frontend) and finally our own wallet inside our app.
```

Comment:
```suggestion
Our team has been building on EOSIO for 4 years now. We've done our pilot in a community in Costa Rica. We've been part of every major announcement since before Block.one released version 1.0. Since then we've deployed and maintained our own private network, developed several features into our app, including [smart contract](https://github.com/cambiatus/contracts), [backend](https://github.com/cambiatus/backend), [infrastructure](https://github.com/cambiatus/backend#nginx-setup), wallet integrations, [frontend](https://github.com/cambiatus/frontend) and finally our own wallet inside our app.
```

Reply:
our pilot community in Costa Rica was the Pulperos app, done in partnership with Fundes, a local ong. Its not ongoing anymore

Reply:
Oops, honest mistake, sorry